configs/components/puppet.rb

```diff
@@ -96,9 +98,26 @@
     pkg.install_file("../osx_paths.txt", "/etc/paths.d/puppet-agent")
   end
 
+  if platform.is_windows?
+    pkg.environment "FACTERDIR" => settings[:prefix]
```

Comment:
This is only needed until https://github.com/puppetlabs/facter/blob/8886a003ac6459e56d78cd3d2d98c6d1e8b1898b/lib/facter.rb.in#L32 is updated. Right now, it doesn't work properly.

cc/ @MikaelSmith